### PR TITLE
Submit GitQL Package

### PR DESCRIPTION
**Pull request by AmrDeveloper**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121198)

### CODE REVIEW

**Comment by AmrDeveloper** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/a/amrdeveloper/gitql/0.7.1/amrdeveloper.gitql.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: amrdeveloper.gitql
+PackageVersion: 0.7.1
+InstallerType: portable
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/AmrDeveloper/GQL/releases/download/0.7.1/gql-x86-windows.exe
+  InstallerSha256: 9A56B5C53B289479157C1AB0D6D8055B61326A05965E108491103082292A9C23
```
Comment (stephengillie) [inline manifests/a/amrdeveloper/gitql/0.7.1/amrdeveloper.gitql.installer.yaml:10]:
```suggestion
  InstallerSha256: 9A56B5C53B289479157C1AB0D6D8055B61326A05965E108491103082292A9C23
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/a/amrdeveloper/gitql/0.7.1/amrdeveloper.gitql.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: amrdeveloper.gitql
+PackageVersion: 0.7.1
+InstallerType: portable
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/AmrDeveloper/GQL/releases/download/0.7.1/gql-x86-windows.exe
+  InstallerSha256: 9A56B5C53B289479157C1AB0D6D8055B61326A05965E108491103082292A9C23
+- Architecture: x64
+  InstallerUrl: https://github.com/AmrDeveloper/GQL/releases/download/0.7.1/gql-x86_64-windows.exe
+  InstallerSha256: 97261a2e8cbde6cbf4c40a9e742226223b62ff9c576ec1683eccc042e55d2da0
```
Comment (stephengillie) [inline manifests/a/amrdeveloper/gitql/0.7.1/amrdeveloper.gitql.installer.yaml:13]:
```suggestion
  InstallerSha256: 97261a2e8cbde6cbf4c40a9e742226223b62ff9c576ec1683eccc042e55d2da0
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by AmrDeveloper** [comment]
@stephengillie Thank you i committed them

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2023-10-01 02:48:47.700 [CORE] Deployment operation #0: Windows cannot install package 28017CharlesMilette.TranslucentTB_2023.1.0.0_x64__v826wp6bftszj because this package depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.8" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 8.2305.5001.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.8" currently installed are: {}

**Review by stephengillie — APPROVED** [review]

**Comment by Dragon1573** [comment]
Close https://github.com/microsoft/winget-pkgs/issues/120711 .

**Pull request merged**